receiver/mysqlreceiver/scraper.go

```diff
@@ -99,10 +99,19 @@ func (m *mySQLScraper) scrape(context.Context) (pmetric.Metrics, error) {
 	m.scrapeIndexIoWaitsStats(now, errs)
 
 	// collect global status metrics.
+	m.scrapeGlobalStats(now, errs)
+
+	m.mb.EmitForResource(metadata.WithMysqlInstanceEndpoint(m.config.Endpoint))
+
+	return m.mb.Emit(), errs.Combine()
+}
+
+func (m *mySQLScraper) scrapeGlobalStats(now pcommon.Timestamp, errs *scrapererror.ScrapeErrors) {
 	globalStats, err := m.sqlclient.getGlobalStats()
 	if err != nil {
 		m.logger.Error("Failed to fetch global stats", zap.Error(err))
-		return pmetric.Metrics{}, err
+		errs.AddPartial(1, err)
```

Comment:
One more nit - can you update this number to match the number of data points we'll miss if the call fails? (62 by my count, but perhaps you can double check me. I counted 4 from `recordDataPages|Usage`)

Reply:
I'm pushing the version with 66 and I'll correct it if you don't agree.